check_source.py

```diff
@@ -77,9 +78,13 @@ def check_source_submission(self, source_project, source_package, source_revisio
             self.review_messages['declined'] = 'Only one action per request allowed'
             return False
 
-        if target_package.startswith('00') or target_package.startswith('_'):
-            self.review_messages['accepted'] = 'Skipping all checks for product related packages'
+        kind = package_kind(self.apiurl, target_project, target_package)
+        if kind == 'meta':
+            self.review_messages['accepted'] = 'Skipping all checks for meta packages'
             return True
+        elif kind != 'source':
+            self.review_messages['accepted'] = 'May not modify a non-source package of type {}'.format(kind)
```

Comment:
@DimStar77 @nilxam @lnussel How often is a multi-spec sub-package overridden with a full source package? The override should still be usable in this case. Or would you rather this check not be added? For example, this would prevent a submit request against `kernel-debug` since it is a multi-spec subpackage of `kernel-source`.

Reply:
We should always decline the *multispec_subpackage* submission in the first place. The only exception here is user want to switch sub-package to be the main-package *+* package update at the same time, but that case happened very rarely, the override for that is good enough.